index.html

```diff
@@ -5497,6 +5506,44 @@ <h2>Base Context</h2>
 }
 </pre>
 
+      </section>
+      <section class="informative appendix">
+        <h2>Differences between Contexts and CredentialSchemas</h2>
+        <p>
+The primary purpose of a <code>credentialSchema</code> is different from the
+primary purpose of <code>@context</code>. A <code>credentialSchema</code> is
+useful for defining the data shape and type of a <code>Verifiable
+Credential</code>, whereas [[JSON-LD]] and a <code>@context</code> in a
+verifiable credential are best used only for conveying the semantics and term 
+definitions of the data.
+        </p>
+
+        <p>
+A <code>@context</code> can help you define [[JSON-LD]] semantics and term
+definitions. A <code>credentialSchema</code> is much more flexible and can help
```

Comment:
I don't understand what "is much more flexible" means. Aren't we trying to highlight that these are for different purposes (semantics vs. shape of data) rather than trying to imply they are for the same thing and one is more flexible than the other?

```suggestion
definitions. A <code>credentialSchema</code> can help
```

Reply:
Yup, I agree with that merging your suggestion.